iceberg/bodo_iceberg_connector/iceberg-java/src/main/java/com/bodo/iceberg/catalog/NessieBuilder.java

```diff
@@ -1,29 +0,0 @@
-package com.bodo.iceberg.catalog;
```

Comment:
Just removed it since we don't support Nessie anyways

Reply:
We did at some point. Any reason to remove it? Does it hurt something?

Reply:
Mainly to avoid confusion for anyone looking at the code and seeing this file. We can always get it back from Git history.